test/image/mocks/gl2d_parcoords_large.json

```diff
@@ -0,0 +1,90 @@
+{
+  "layout": {
+    "width": 1184,
+    "height": 400,
+    "title": "Widget parameters and cost",
+    "margin": {"t": 80, "r": 80, "b": 80, "l": 80}
```

Comment:
A better test mock would have `margin.r`, `margin.b` and `margin.l` not equal to their default values.